### DISCUSSION TITLE
Helping out fellow lumenauts

### DISCUSSION THREAD

**Discussion by hanseartic** [stellar/sorobanathon#161] [General]
This is probably not exactly the too fancy contribution but besides my own submission I chimed in here and there:

<details><summary>Pull requests</summary>
<p>

### other ***Sorobanathon: First Light 🔭***  submissions
* https://github.com/vinamogit/soroban-rand/pull/1

### ~~Stellar~~ Soroban Quest
* https://github.com/tyvdh/soroban-quest/pull/7
* https://github.com/tyvdh/soroban-quest--pioneer/pull/12
* https://github.com/tyvdh/soroban-quest--pioneer/pull/13
* https://github.com/tyvdh/soroban-quest--pioneer/pull/15
* https://github.com/tyvdh/soroban-quest--pioneer/pull/20
* https://github.com/tyvdh/soroban-quest--pioneer/pull/21
* https://github.com/tyvdh/soroban-quest--pioneer/pull/22
* https://github.com/tyvdh/soroban-quest--pioneer/pull/23

</p>
</details>

**Comment by kalepail** [comment]
Thanks again for all your incredible help with Stellar Quest! Great idea submitting these to the Sorobanathon!

**Reply by hanseartic** [reply]
Thank you for having me 🤗

**Comment by achallagundla** [comment]
Heya,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1000 XLM (a small token of gratitude for all your help with SQ)! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
